src/Core/Prefab/GlobeView.js

```diff
@@ -371,10 +371,17 @@ GlobeView.prototype.readDepthBuffer = function readDepthBuffer(x, y, width, heig
 
 const matrix = new THREE.Matrix4();
 const screen = new THREE.Vector2();
-const pickWorldPosition = new THREE.Vector3();
 const ray = new THREE.Ray();
 const direction = new THREE.Vector3();
-GlobeView.prototype.getPickingPositionFromDepth = function getPickingPositionFromDepth(mouse) {
+
+/**
+ * returns the pick position under the mouse
+ *
+ * @param      {THREE.Vector2}  mouse   mouse coordinates to pick position
```

Comment:
What coordinates systems? Is this geographic coordinates? Or normalized? Or pixels?